src/core/classes/language-registry.ts

```diff
@@ -75,6 +79,7 @@ export default class LanguageRegistry extends ComponentRegistry<LanguageLike> {
 	 * Get resolved language or null if it doesn't exist
 	 * If definition is loaded but not yet resolved, it will be resolved and returned.
 	 */
+	// FIXME: This method overrides the get() method from the base class (ComponentRegistry), but it has a different signature. TS complains about it.
```

Comment:
Ooh I forgot there was a base `get()`, we should name it something else I guess. Maybe `resolve()`?